src/common/exif.cc

```diff
@@ -1110,6 +1110,19 @@ int dt_exif_write_blob(uint8_t *blob, uint32_t size, const char *path, const int
       dt_remove_exif_keys(imgExifData, keys, n_keys);
     }
 
+    // remove the profile tables
```

Comment:
I was thinking that maybe it would be useful to have access to this information if somebody wanted to implement using curves provided in the raw files?